test/resources/CalendarExporterTest/testMultipleResourcesExport.OUT.1.iCal

```diff
@@ -0,0 +1,54 @@
+BEGIN:VCALENDAR
+VERSION:2.0
+PRODID:https://oerworldmap.org/
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+BEGIN:VEVENT
+SUMMARY:DigiLern 2012
+ORGANIZER;CN="Ottobrunn Grammar School":mailto:sekr@gymnasium-ottobrunn.de
+DTSTART:20120308
```

Comment:
This is also done in https://github.com/hbz/oerworldmap/commit/3037a18ccacf3591959d3b1ec0a2f0a004e64eb7.